tests/tizen_capi/unittest_tizen_sensor.cc

```diff
@@ -304,24 +299,15 @@ TEST (tizensensor_as_source, virtual_sensor_flow_04)
   EXPECT_NE (state, ML_PIPELINE_STATE_NULL);
 
   count = 0;
-  while (state != ML_PIPELINE_STATE_PLAYING) {
-    g_usleep(10000); /* 10ms */
-    status = ml_pipeline_get_state (handle, &state);
-    EXPECT_EQ (status, ML_ERROR_NONE);
-    count++;
-    EXPECT_LE (count, 50);
-    if (count >= 50)
-      break;
-  }
+  wait_for_start (handle, state, status);
+  EXPECT_EQ (state, ML_PIPELINE_STATE_PLAYING);
+
   g_usleep(10000); /* Let a frame or more flow */
   value.values[0] = 1.01;
   EXPECT_EQ (dummy_publish (sensor, value), 0);
 
-  g_usleep (100000); /* 100ms. Let a few frames flow. */
-  status = ml_pipeline_get_state (handle, &state);
-  EXPECT_EQ (status, ML_ERROR_NONE);
-  EXPECT_TRUE (state == ML_PIPELINE_STATE_PLAYING ||
-      state == ML_PIPELINE_STATE_PAUSED);
+  EXPECT_TRUE (wait_pipeline_process_buffers (&data.checked, 2, TEST_TIME_OUT_TIZEN_SENSOR));
+
   EXPECT_GT (data.checked, 1);
```

Comment:
LINE 311 not needed.

Reply:
Yes! `wait_pipeline_process_buffers` handles this line.